### PR TITLE
Replace all non-klog logging lines with klog

### PR DESCRIPTION
**Pull request by courtneypacheco**
The operator code is inconsistent in terms of the types of logging methods used. Prior to this commit, logr, logf, etc. were being
used in addition to klog. To keep things consistent, I have removed all non-klog logging statements and replaced them with klog logging statements.

### CODE REVIEW

**Review by ArangoGutierrez — APPROVED** [review]
waiting on the extra feature we discussed offline, but looks good so far , thanks for this

**Comment by ArangoGutierrez** [comment]
after this PR, TODO should be klog/v2

File: go.mod [hunk 1]
```diff
@@ -12,7 +12,7 @@ require (
 	k8s.io/api v0.20.4
 	k8s.io/apimachinery v0.20.4
 	k8s.io/client-go v0.20.4
-	k8s.io/klog v1.0.0
+	k8s.io/klog/v2 v2.4.0
```
Comment (ArangoGutierrez) [inline go.mod:15]: Can it be 2.9 as NFD https://github.com/kubernetes-sigs/node-feature-discovery/blob/master/go.mod#L20

**Review by ArangoGutierrez — CHANGES_REQUESTED** [review]

**Comment by courtneypacheco** [comment]
@ArangoGutierrez I changed the klog version to v2.9.0 as requested. PTAL.

**Comment by ArangoGutierrez** [comment]
Testing this one today

**Review by ArangoGutierrez — APPROVED** [review]
looks good, will merge once we push this to upstream first

**Comment by ArangoGutierrez** [comment]
Rebase after https://github.com/openshift/cluster-nfd-operator/pull/184 merge

**Review by ArangoGutierrez — APPROVED** [review]
/lgtm

**Pull request merged**